docs/configurationInstructions.md

```diff
@@ -104,39 +104,6 @@ data = "LS0tLS1CRUdJTiBFTkNSWVBURUQgUF…JVkFURSBLRVktLS0tLQo="
 
 See [here](../docs/companions.md) how to configure companions.
 
-## Hooks
```

Comment:
We should keep the headline and point to the hooks files as the document does for companions.